client/state/data-layer/wpcom/activity-log/update-credentials/index.js

```diff
@@ -134,8 +134,8 @@ export const failure = ( action, error ) => ( dispatch, getState ) => {
 	const spreadHappiness = ( message ) => {
 		const tracksEvent = recordTracksEvent( 'calypso_rewind_creds_update_failure', {
 			site_id: action.siteId,
-			error: error.error,
-			status_code: error.statusCode,
+			error: error.code,
+			status_code: error.data ?? error.statusCode,
```

Comment:
What kind of information does `error.data` provide?

Reply:
in general, WP_Error->data can contain anything (class-wp-error.php:37), but in this case it’s an HTTP status code (class.wpcom-json-api-rewind-update-credentials-endpoint.php:77)

Reply:
it’s pretty funny if you think about it… by the time an invalid_credentials error reaches calypso, it’s a 403 wrapped in a 500 wrapped in a 200 c;